frontend/src/locales/scripts/en.json5

```diff
@@ -498,6 +498,11 @@
     },
     imageInfo: "Image information",
     audioInfo: "Audio information",
+    tags: {
+      title: "Tags",
+      showMore: "Show more",
+      showLess: "Show less",
```

Comment:
Grammar nit: While "Show less" feels right too, I'm feeling a little bit of a dissonance that says this should be "Show fewer" because tags are countable.

Reply:
That's "show more content" and "show less content" 😆